packages/components/src/components/list-box/_list-box.scss

```diff
@@ -343,6 +343,7 @@ $list-box-menu-width: rem(300px);
   .#{$prefix}--list-box__selection {
     position: absolute;
     right: rem(33px); // to preserve .5rem space between icons according to spec
+    top: 0.35rem;
```

Comment:
Do we need to use bespoke positioning or could we do `top: 0; bottom: 0;` to center?

Reply:
let me check 👍

Reply:
Seems like we settled on `top: 50%;` `bottom: translateY(-50%);` 👍